site/index.html

```diff
@@ -59,6 +90,17 @@ <h3>Haiku</h3>
                 <cite>&mdash; <span class="author">Friedrich Nietzsche</span></cite>
             </blockquote>
             <p>&copy; 2025 <a href="mailto:luisfelipedavilap@gmail.com?subject=Message%20%from%CSD112%20Website">Luis Felipe</a></p>
+
+            <section>
+                <picture>
+                    <source srcset="images/sc-logo/sc-logo.svg" type="image/svg+xml">
```

Comment:
5. `Lab 3 > Working with Image Support > picture element > svg has width of 70px`
```suggestion
                    <source srcset="images/sc-logo/sc-logo.svg" type="image/svg+xml" width="70">
```